lib/filter_validator.rb

```diff
@@ -29,14 +29,20 @@ def valid_element?(result, elements, iterations = [])
     end
 
     if method.match(ALL)
-      Array.wrap(result).each.with_index.all? do |x, index|
+      return false unless result.is_a?(Array)
```

Comment:
If I'm not mistaken, this is a change in behaviour, since the `wrap` could have ended with a truthy result. Maybe a desired change, but we _might_ break things, unless I'm missing something? I should probably open my Mac and test things out...